sdk/guestconfiguration/Microsoft.Azure.Management.GuestConfiguration/src/Microsoft.Azure.Management.GuestConfiguration.csproj

```diff
@@ -6,7 +6,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.GuestConfiguration</PackageId>
     <Description>Provides Microsoft Azure Guest Configuration management operations including the ability to create, update and get guest configuration assignments, compliance statuses, guest configuration assignment reports.</Description>
-    <Version>1.5.0</Version>
+    <Version>1.6.0</Version>
```

Comment:
As interface changes caused not backward compatible, please increate the major version of the package, not minor version

Reply:
done